server/container/cli/src/test/java/org/apache/james/cli/ServerCmdTest.java

```diff
@@ -1404,18 +1403,22 @@ public void getPortShouldThrowOnTooHighPortValueOption() throws Exception {
     }
 
     @Test
-    public void uploadAndActivateScript() throws Exception {
+    public void addActiveSieveAndGetActiveScript() throws Exception {
```

Comment:
Could you add a test when the file has:

 - The .siv extension?
 - Nether the .sieve nor .siv extension?